FIAT/macro.py

```diff
@@ -390,11 +391,40 @@ def __init__(self, ref_el, degree, order=1, shape=(), **kwargs):
         if len(rows) > 0:
             dual_mat = numpy.vstack(rows)
             _, sig, vt = numpy.linalg.svd(dual_mat, full_matrices=True)
-            num_sv = len([s for s in sig if abs(s) > 1.e-10])
+            tol = sig[0] * 1E-10
+            num_sv = len([s for s in sig if abs(s) > tol])
             coeffs = vt[num_sv:]
         else:
             coeffs = numpy.eye(expansion_set.get_num_members(degree))
 
+        if vorder > order + 1:
```

Comment:
Or is supersmoothness only implie for order + 1?  A citation/comment would be helpful